agent/command/assume_ec2_role.go

```diff
@@ -120,9 +120,9 @@ func (r *ec2AssumeRole) Execute(ctx context.Context,
 		return errors.WithStack(err)
 	}
 
-	conf.Expansions.Put(AWSAccessKeyId, credValues.AccessKeyID)
-	conf.Expansions.Put(AWSSecretAccessKey, credValues.SecretAccessKey)
-	conf.Expansions.Put(AWSSessionToken, credValues.SessionToken)
-	conf.Expansions.Put(AWSRoleExpiration, expTime.String())
+	conf.NewExpansions.Put(AWSAccessKeyId, credValues.AccessKeyID)
```

Comment:
do we need to `Put` into both?

Reply:
No because it is wraps the same underlying map so doing it once is ok. I think eventually we would want to move everything in the agent to just use the the new `DynamicExpansions`.